packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.base.tsx

```diff
@@ -678,10 +656,7 @@ export class DropdownBase extends BaseComponent<IDropdownInternalProps, IDropdow
   };
 
   // Get all selected indexes for multi-select mode
-  private _getSelectedIndexes(
-    options: IDropdownOption[],
-    selectedKey: string | number | string[] | number[] | undefined
-  ): number[] {
+  private _getSelectedIndexes(options: IDropdownOption[], selectedKey: string | number | string[] | number[] | undefined): number[] {
     if (selectedKey === undefined) {
```

Comment:
Are these changes from Prettier?

Reply:
When I run save on my machine, these are not changed. As such, these are probably regressive.

Do you have prettier extension installed? If not, how are you committing these files? (in theory it should format for you when you commit)

---
In reply to: [220036854](https://github.com/OfficeDev/office-ui-fabric-react/pull/6462#discussion_r220036854) [](ancestors = 220036854)

Reply:
That is how it formatted on saving the file... so I assumed it is. Unless there is something wrong with my code editor. Should I have the Prettier plugin disabled and leave the pre-commit hooks to do their job?

Reply:
@cliffkoh I tried reversing the commit and disabled the Prettier plugin for this particular workspace and tried to commit again and it did the same thing. I believe there was a change in max-line lenght with this #6246 and this might have been the cause of aggressive formatting. Before was 120 now it's 140. I tried put in a new branch the changed lines in one line and they all come up in between 120 and 140... so it looks like it's doing it's job when not reaching 140 to put the expression in one line. I suppose we should expect more changes like this in the future PRs.